### DISCUSSION TITLE
Zoomed right sidebar inconsistently shown in Safari

### DISCUSSION THREAD

**Discussion by mtdowling** [pradyunsg/furo#483] [Potential Issue]
When using Safari (Version 15.5 (17613.2.7.1.8 on Mac OS 12.4), the right sidebar isn't always visible when it should be. You can observe this on the stock Furo documentation by zooming to the point where the side is just about to disapear due to the reduced available size. With the sidebar still visible, refresh the page, and it will disappear. Note that I can't reproduce this in FireFox.

**Comment by pradyunsg** [comment]
Can you share a video of this reproducer?

**Reply by pradyunsg** [reply]
Closing due to a lack of a response.
